src/services/ansibleLint.ts

```diff
@@ -151,6 +159,11 @@ export class AnsibleLint {
       if (progressTracker) {
         progressTracker.done();
       }
+    } else {
+      console.debug(
+        'Ansible-lint is disabled. Falling back to ansible syntax-check'
+      );
+      return;
```

Comment:
It'd be much easier to follow if you reversed the if-clause and turned it into a "guard expression". This would allow you to effectively put the whole block above into an outer scope.
```ts
if (!settings.ansibleLint.enabled) {  // put this in front of everything to exit early
  console.debug(
    'Ansible-lint is disabled. Falling back to ansible syntax-check'
  );
  return;
}

// that whole block from above here, outside of {}
```

Reply:
Thanks for the suggestion.